deploy_pulp3/README.md

```diff
@@ -0,0 +1,6 @@
+1. install ansible2.2+, python3, virtualenv, libselinux-python
```

Comment:
Which virtualenv package should users install? python2-virtualenv? python3-virtualenv? Fedora 25 has no virtualenv package.

Reply:
I believe python3-virtualenv should be used for pulp3

Reply:
Hunh. I tried that, but the playbook failed to run.